crawlfrontier/core/manager.py

```diff
@@ -324,13 +335,13 @@ def get_next_requests(self, max_next_requests=0):
                                             (max_next_requests, self.n_requests, self.max_requests or '-')))
 
         # get next requests
-        next_requests = self.backend.get_next_requests(max_next_requests)
+        next_requests = self.backend.get_next_requests(max_next_requests) or []
```

Comment:
What's the reason for the type diversity here? May be it would be more readable if backend method get_next_requests will be allowed only to return list? Empty one, if there is no results.